README-zh.md

```diff
@@ -9,7 +9,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.uber:okbuck:0.54.1'
```

Comment:
This is referring to the previously public release. SO this should be .1 or .2 until we do the actual cut of this new code.

This is the normal flow:
Step 1. Land your PR without any version changes (and optional changelog updates). 
Step 2: Create a Release PR (move version numbers in readmes and property files are set) and you tag
Step 3: Cut release to maven central and github
Step  4: Create PR that resets -SNAPSHOT url to get ready for new commits.

Check the prior version git version history for this pattern.

Or if you're working on a repo like ribs, it's all rolled up into the releasing.sh script.